### ISSUE TITLE
No router instance found after working on a different React project

### ISSUE THREAD

**Issue by michael-stubbs** [vercel/next.js#26234]
### What version of Next.js are you using?

10.2.3

### What version of Node.js are you using?

14.16.1

### What browser are you using?

Firefox

### What operating system are you using?

Windows

### How are you deploying your application?

Localhost

### Describe the Bug

After working on a separate react project (in a completely separate folder), swapping back to my Next.js project and running `npm run dev` results in `Error: No router instance found.` and `Error: Invalid hook call`. 

I have confirmed that I stopped the previous dev environment before starting a new one.

The app has not been changed and matches the remote version. It will work fine before, but as soon as I work on the other project in a dev server (same Node version) it breaks mine. The other project is running an older version of react.

I am also spammed with `There are multiple modules with names that only differ in casing.` when I try to run my Next.js project.

These are completely separate file systems and projects, why would they interact like this? For example, I could go pull the exact same version of this project on my laptop and it would work fine.

Next.js project repo: https://github.com/michael-stubbs/doki-design

### Expected Behavior

The Next.js app does not interact with other React projects in other directories.

### To Reproduce

Work on an older React version project in localhost, then switch to a Next.js project with a newer React version.

**Comment by ijjk** [comment]
Hi, it sounds like `next` or `react` might be being installed globally causing conflicts with the project. Please ensure none of these dependencies are installed globally and also ensure this isn't occurring in the latest version of Next.js `v11.1.2`

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
